apps/backoffice-v2/src/pages/Entity/components/CallToAction/CallToAction.tsx

```diff
@@ -45,23 +58,34 @@ export const CallToAction: FunctionComponent<ICallToActionProps> = ({ value, dat
     onCommentChange,
     noReasons,
     workflowLevelResolution,
-  } = useCallToActionLogic();
+    isReuploadResetable,
+  } = useCallToActionLogic({
+    contextUpdateMethod,
+    revisionReasons,
+    rejectionReasons,
+    onReuploadReset,
+  });
+
+  const handleDialogClose = useCallback(
+    (open: boolean) => {
+      if (!open) onDialogClose && onDialogClose();
+    },
+    [onDialogClose],
+  );
 
-  if (value === 'Reject') {
+  if (value?.text === 'Reject') {
     return (
-      <Dialog>
+      <Dialog onOpenChange={handleDialogClose}>
```

Comment:
Should this function be named `handleDialogChange`?

Reply:
It could but if it handled not just `close` state